app/src/main/java/org/mozilla/tv/firefox/pocket/PocketVideoFetchScheduler.kt

```diff
@@ -53,6 +53,19 @@ class PocketVideoFetchScheduler(
             .setRequiredNetworkType(NetworkType.CONNECTED)
             .build()
 
+        val delay: Long
+        val workPolicy: ExistingWorkPolicy
+
+        // This sets the custom fetch delay for testing purposes.
+        // The custom delay will only run once.
```

Comment:
nit: I'd put this comment above the values so we can better group the code together without misleading new lines.